clientv3/watch.go

```diff
@@ -28,6 +28,7 @@ import (
 const (
 	EventTypeDelete = mvccpb.DELETE
 	EventTypePut    = mvccpb.PUT
+	EventTypeExpire = mvccpb.EXPIRE
```

Comment:
Actually there is no expire event right now. We probably should remove this from the proto file.